tests/BenchmarkDotNet.IntegrationTests/TailCallDiagnoserTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Configs;
+#if NET46
+using BenchmarkDotNet.Diagnostics.Windows;
+using BenchmarkDotNet.Diagnostics.Windows.Configs;
+using BenchmarkDotNet.Environments;
+#endif
+using BenchmarkDotNet.IntegrationTests.Xunit;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Tests.Loggers;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace BenchmarkDotNet.IntegrationTests
+{
+#if NET46
+    public class TailCallDiagnoserTests : BenchmarkTestExecutor
```

Comment:
Am I right that these tests should work only on Windows?
`#if NET46` requires only full .NET Framework (not .NET Core), but it's a valid profile for Mono+Linux/macOS.
If you have tests which should work only on Windows, please use `TheoryWindowsOnlyAttribute`
https://github.com/dotnet/BenchmarkDotNet/blob/696f89908e82118ecf9989c2484c23a6f52ba6b2/tests/BenchmarkDotNet.IntegrationTests/Xunit/TheoryWindowsOnlyAttribute.cs

Reply:
True, it's working only for windows with full .NET Framework, thus I may use `TheoryWindowsOnlyAttribute` plus `#if NET46` to fix compile time errors, related to `BenchmarkDotNet.Diagnostics.Windows.Configs` which does not support .NET CORE. Does it make sense?

Reply:
Yes.